src/screens/AddressDetail.js

```diff
@@ -263,13 +270,23 @@ class AddressDetail extends React.Component {
     });
   }
 
+  getSelectedTokenMetadata = (selectedToken) => {
+    metadataApi.getDag(selectedToken).then((data) => {
```

Comment:
`getDag`? I feel we should rename it to `getDagMetadata()`.

Reply:
Makes sense. Changed that